go/exocom/src/cmd/index.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "exocom",
+	Short: "",
+	Run: func(cmd *cobra.Command, args []string) {
+		port := os.Getenv("PORT")
+		if port == "" {
+			port = "3100"
+		}
+		fmt.Printf("ExoCom online at port %s\n", port)
+		err := http.ListenAndServe(fmt.Sprintf(":%s", port), nil)
+		if err != nil {
+			panic(err)
```

Comment:
Not sure if we want to panic here. Panics are pretty rare in Go. If there is an error here, let's exit cleanly using fmt.Fatal.